src/components/alert/alert.html

```diff
@@ -0,0 +1,217 @@
+					<span class="m_doc_class-item">alert</span>
+					<span class="m_doc_class-item">alert--primary</span>
+					<span class="m_doc_class-item">alert--secondary</span>
+					<span class="m_doc_class-item">alert--info</span>
+					<span class="m_doc_class-item">alert--success</span>
+					<span class="m_doc_class-item">alert--danger</span>
+					<span class="m_doc_class-item">alert--warning</span>
+					<span class="m_doc_class-item">alert--dismissable</span>
+				</span>
+			</p>
+
+			<!-- component display container -->
+			<div class="component_display_container flex-center margin-top">
+				<div class="alerts flex-col-align-center fw">
+					<div class="alert alert--primary">
+						Primary! This is a primary alert—check it out!
+					</div>
+					<div class="alert alert--secondary">
+						Secondary! This is a secondary alert—check it out!
+					</div>
+					<div class="alert alert--info">
+						Info! This is a info alert—check it out!
+					</div>
+					<div class="alert alert--success">
+						Success! This is a success alert—check it out!
+					</div>
+					<div class="alert alert--danger">
+						Danger! This is a danger alert—check it out!
+					</div>
+					<div class="alert alert--warning">
+						Warning! This is a warning alert—check it out!
+					</div>
+				</div>
+			</div>
+
+			<!-- code snippet container -->
+
+			<div class="snippet_container">
+				<script
+					type="text/plain"
+					class="line-numbers language-css snippet_container__script"
+				>
+				<div class="alert alert--primary">
+					//Your Alert text goes here..
+				  </div>
+				  <div class="alert alert--secondary">
+					//Your Alert text goes here..
+				  </div>
+				  <div class="alert alert--info">
+					//Your Alert text goes here..
+				  </div>
+				  <div class="alert alert--success">
+					//Your Alert text goes here..
+				  </div>
+				  <div class="alert alert--danger">
+					//Your Alert text goes here..
+				  </div>
+				  <div class="alert alert--warning">
+					//Your Alert text goes here..
+				  </div>
+				  
+				</script>
+			</div>
+
+
+			<!-- Dismissible Alert component -->
+			<h3>Dissmissable Alerts</h3>
+
+			<p class="m_doc-para">
+				Dissmissable Alerts can be created by adding following to standard
+				Alerts.
+				<span class="m_doc_class">
+					<span class="m_doc_class-item">alert--dismissable</span>
+					<span class="m_doc_class-item">alert--btn__dismiss</span>
+				</span>
+			</p>
+
+			<!-- component display container -->
+			<div class="component_display_container flex-center margin-top">
+				<div class="alerts flex-col-align-center fw">
+					<div id="alert--warning--dismiss" class="alert alert--dismissable">
+						This is a dismissable warning alert! (add width:fit-content to
+						reduce the size of the alert)
+						<button
+							id="alert--warning--dismiss-btn"
+							class="btn btn--icon btn--close--transparent alert--btn__dismiss btn--circular"
+						>
+							<i class="bx bx-x"></i>
```

Comment:
Confusing naming of 'bx' , 'bx-x'
Can improve these.

Reply:
These are of BOXICON font icons.